### ISSUE TITLE
Issue: editing a plan retention policy may turn page blank

### ISSUE THREAD

**Issue by modem** [garethgeorge/backrest#91]
Hi Gareth,

this one is weird. I was editing a plan, and was changing from the default retention policy 30 count. I did the following steps: 

1. placed the cursor after the `30` value
2. pressed `Backspace`2 times to delete the number:
    - the 1st key press deletes the 0
    - the 2nd key press turns the full backrest page blank.

I then need to refresh the page for the content to appear again.

Placing the cursor before the number, and pressing the `DELETE` key twice has the same behavior.

I have the same behavior on Firefox and Edge.

**Comment by modem** [comment]
I see you changed the behavior, but not exactly what I was expecting.
On the new behavior, when the entire count value is deleted, the **None** policy is automatically selected, but as per my use, when I delete the entire count I want to replace by a different one. Maybe an error should be visible when the count is empty (or invalid) instead of selecting something else.

**Comment by dhilgarth** [comment]
@garethgeorge Maybe, the correct approach here would be one where the state is only updated on blur, not on keypress/keyup?

**Comment by garethgeorge** [comment]
I've started drafting a fix in https://github.com/garethgeorge/backrest/tree/garethgeorge/prunepolicyfix . there are some issues with how I structured the prune policy in the config which make it awkward to determine which state it should be in. I'm going to migrate it to a oneof to make this more explicit. 

The next release is primarily focused on reliability and new platform support and, while this is a bug fix, it comes with some risk as it introduces a config migration so I'm going to hold off until 0.14.0 to patch this.
